CRM/Contact/BAO/Group.php

```diff
@@ -1119,11 +1120,12 @@ private static function buildGroupHierarchy(&$hierarchy, $group, $tree, $titleOn
       $hierarchy[$group['id']] = $spaces . $group['title'];
     }
     else {
-      $hierarchy[$group['id']] = [
-        'title' => $spaces . $group['title'],
+      $hierarchy[] = array(
```

Comment:
@monishdeb is there a reason why we are changing the array key here? also shouldn't this use the short array syntax?

Reply:
@seamuslee001 the answer lies in [here](https://github.com/civicrm/civicrm-core/pull/13958/files#diff-e8f2bd059a6e0b2434434982052bfca9R59) where we are now relying on select2 widget, which expects the array keys to be sequential and not hardcoded group ID. And better coverage for tests

Reply:
Could you update for the short array syntax?